content/workload-reduction-toolkit/address-workload-issues/curriculum-planning-and-delivery/try-ideas-to-manage-planning.md

```diff
@@ -0,0 +1,87 @@
+layout: /markdown_resource.html.erb
+---
+
+# Try ideas to manage planning and curriculum workload
+
+<strong class="govuk-tag">Case study</strong>
+
+{inset-text}
+
+## School details
+
+**School name:** King Charles I School
+
+**Location:** Worcestershire
+
+**Phase:** Secondary
+
+**Number of pupils:** 1000
+
+**Contact details:** Email deputy headteacher Ruth Allen at  <rallen@kingcharles1.worcs.sch.uk>
+
+{/inset-text}
+
+<div class="govuk-grid-row dfe-width-container">
+  <div class="govuk-grid-column-full">
+    <div class="info-box">
+      <div class="info-box__corner">
+        <img src="/assets/images/bullseye.svg" alt="Bullseye icon">
+      </div>
+      <h2 class="govuk-heading-m">
+        Impact and outcomes
+      </h2>
+      <p>
+        We made these changes to enable every teacher to become highly effective by:
+      </p>
+      <ul>
+        <li>
+        ensuring every teacher has time to focus on what is important - planning, teaching and feedback
+        </li>
+        <li>
+        believing in simplicity, always taking the shortest route and aiming for maximum impact on student learning with minimal workload for staff 
+        </li>
+        <li>
+        continuously reviewing and evaluating our systems in order to support all staff to achieve a healthy work life balance
+        </li>
+     </ul> 
+      </p>
+    </div>
+  </div>
+</div>
+
+## Background from Ruth Allen, Deputy Headteacher
+
+{inset-text}
+
+At King Charles I school we know that teaching is exhilarating and rewarding but recognise that it can also be exhausting. We understand that time is precious and that tired teachers do not make effective teachers. Our aim is for staff to not have to take work home. 
+{/inset-text}
+
+## 15 ways that we manage curriculum planning and delivery workload 
+
+Collaborative planning enables teachers to discuss and share best practice rather than everyone busily planning the same lessons in their individual classrooms. We accept that collaborative planning will lead to less autonomy in the classroom but the trade off is much less work to do at home. 
+
+Staff within departments, where possible, have preparation, planning and assessment time (PPA) together to allow for collaborative planning. 
+
+Departmental resources are shared centrally to stop individuals having to reinvent the wheel with new plans. 
+
+We provided better quality schemes of work that specify the sequence of key knowledge, deliberate practice to develop skills, a clear progression model, alongside homework tasks and resources to be used in lessons. This means there is no more endless searching for online resources for single activities, homework tasks and lesson ideas. 
+
+All teachers follow research driven teaching models that focus on activities that have a significant impact on progress. No more time spent creating time wasting teaching resources such as card sorts and slideshows.   
```

Comment:
Just rewording slightly so the second line in particular reads more like a complete sentence - "All teachers follow research-driven teaching models that focus on activities that have a significant impact on progress. In our school, we do not waste time creating teaching resources such as card sorts and slideshows."